Libs/MRML/Core/vtkMRMLViewNode.h

```diff
@@ -334,14 +334,12 @@ class VTK_MRML_EXPORT vtkMRMLViewNode : public vtkMRMLAbstractViewNode
   /// to object size of about 100 (in scene physical units).
   vtkGetMacro(AmbientShadowsSizeScale, double);
   vtkSetMacro(AmbientShadowsSizeScale, double);
-  vtkBooleanMacro(AmbientShadowsSizeScale, double);
```

Comment:
For reference, the use of `vtkBooleanMacro` was originally introduced in https://github.com/Slicer/Slicer/commit/02627b5f6d354f003bba21b4ec2d2e4dfef28f4e